### PR TITLE
Add big green button

### PR DESCRIPTION
**Pull request by sjpfenninger**
Other minor additions/changes:

* Reduced size of guru logo to increase prominence of new button
* Added Gemfile

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -2,10 +2,16 @@
...
-<img style="display: block; margin: 0 auto" src="./images/logo.png" alt="CSP.guru Logo" align="center">
+<img style="display: block; margin: 0 auto" width="200" src="./images/logo.png" alt="CSP.guru Logo" align="center">
 
 {% assign description = site.data.datapackage["description"] %}
 
 {{ description }}
 
-Get it here: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1318152.svg)](https://doi.org/10.5281/zenodo.1318152)
+<p style="padding: 20px 0; text-align: center;">
+    <a class="button green" href="https://zenodo.org/record/1318152/files/repolicy/csp-guru-2016-08-30.zip?download=1">Direct data download</a>
```
Comment (timtroendle) [inline index.md]: Would be nice to have a link to `latest` somehow here, but that might not be possible.

There is a link to the page of the latest version on Zenodo:
https://doi.org/10.5281/zenodo.1318151

But I'm not sure whether there is a link to the download as well.

**Comment by timtroendle** [comment]
@richard-thonig any comments?

File: index.md (hunk 1 above)
Comment (sjpfenninger) [reply index.md]: I agree, but I also don't know whether it is possible.

**Comment by richard-thonig** [comment]
Thank you so much Stefan. It is a very nice button.

**Pull request merged**